source/_integrations/thread.markdown

```diff
@@ -69,7 +69,11 @@ Thread uses the same <abbr title="radio frequency">RF</abbr> technology as Zigbe
 
 ### About Thread border routers
 
-The devices use Thread border routers to communicate outside the mesh with any IPv6-capable device. A Thread border router is connected to your network either via Wi-Fi or Ethernet and uses its <abbr title="radio frequency">RF</abbr> radio to communicate with the Thread mesh network. The <abbr title="Thread border router">TBR</abbr> routes packets between your local network and the Thread mesh. It does not look at the content of these packets, it just forwards them.
+A Thread border router is a network device that connects a Thread mesh network (consisting of low-power IoT devices) to other IP networks such as Ethernet or Wi-Fi.
```

Comment:
"consisting of low-power IoT devices" is not wrong, but I'd then mention they have to be Thread devices.
I'm not sure we even have to be that verbose though. Maybe something more simple works as well? E.g.:

```suggestion
A Thread border router is a device that connects a Thread mesh network to your local network via Wi-Fi or Ethernet.
```
or
```suggestion
A Thread border router is a device that connects a Thread mesh network to your home's IP network via Wi-Fi or Ethernet.
```


It uses a radio chip to communicate with low-power Thread devices in the mesh, and routes their traffic to and from your local network.
The Thread border router does not control devices. Device control is handled by application layer protocols like Matter or Apple HomeKit.